commcare_export/env.py

```diff
@@ -222,9 +222,17 @@ def emitted_tables(self):
 
 @unwrap('val')
 def str2bool(val):
+    if val is None:
+        return False
+
     if isinstance(val, bool):
         return val
-    return val and str(val).lower() in {'true', 't', '1'}
+
+    if not isinstance(val, six.string_types):
+        val = str(val)
```

Comment:
Thinking a bit more about "We should probably use this everywhere..." In the context of `str2bool`, `val.decode('ascii', 'replace')` will work just fine, but in other contexts we may want to decode bytes using some other method, possibly `val.decode('utf-8', 'replace')`?